hydrant/views.py

```diff
@@ -99,8 +99,26 @@ def export(adapter, filter):
         next_page_link = jmespath.search('link[?relation==`next`].{url: url}', bundle)
         if not next_page_link:
             break
-        response = requests.get(next_page_link[0]['url'])
-        bundle = response.json()
+
+        # HAPI server config uses a `dashboard` protected URL, which we can't write to.
+        #   see: https://github.com/uwcirg/cosri-environments/pull/56
+        # correct server portion of `next` to configured `FHIR_SERVER_URL`,
+        # relying on `/fhir` request path in both
```

Comment:
maybe add a TODO for when we can use the `next_page_link` as-is returned by HAPI?